src/__tests__/VillainIndex.test.js

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from "@testing-library/react"
+import VillainIndex from "../pages/VillainIndex"
+import mockVillains from "../mockVillains";
+
+describe("<VillainIndex />", () => {
+    it("renders without crashing", () => {})
+    it("renders villain cards", () => {
+        const div = document.createElement("div")
+        render(<VillainIndex mockVillains={mockVillains} />, div)
+        mockVillains.forEach(villain => {
+            const villainName = screen.getByText(villain.name)
+            expect(villainName).toBeInTheDocument()
+        })
+    })
+})
+
+
+//NOT currently passing - does not get name from page (line 11)
```

Comment:
Do you need help with this blocker? If not, then please use reviews for working code. Please feel free to come out and discuss this blocker.